README.md

```diff
@@ -1,56 +1,58 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# REACT/REDUX Project - Metrics webapp
 
+> In this porject, I worked with the data fetch from the financialmodelingprep API. I build a web application to get information about the companies listed in stock exchange. It is build with React/Redux and fetch data using API.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- HTML/CSS
+- Javascript
+- React/Redux
+- Linters
+- Visual Studio
+- Git/GitHub
+- GitFlow
+- Webpack
+- Jest Testing
+- Netlify
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+[Live Demo](https://stock-metrics-webapp.netlify.app/)
+[Video Demo](https://www.loom.com/share/d816f2ac60b042b0b0c360c7a366e2a3)
```

Comment:
- [ ] Nice presentation, in the video please add a piece of code that you're most proud of as stated in the requirements.

![Screen Shot 2022-09-02 at 1 02 37 PM](https://user-images.githubusercontent.com/59416136/188115888-0fa747a5-35a6-40e3-af91-858cac718890.png)

Reply:
@Wes-Isaac,
Thanks for the review
- Updated Readme to add Acknowledgement of the designer
- Also recreate the video with highlight the section on which I am proud of.